config/crd/bases/operator.kyma-project.io_kcpmodules.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.13.0
```

Comment:
Is it safe to update kubebuilder version "in-place" or should we rather have a dedicated PR to update that (also applies to the other CRDs)?

Reply:
that is actually a downgrade because I did not update my controller-gen binary. will fix